server/telemetry/model.go

```diff
@@ -0,0 +1,26 @@
+package telemetry
+
+import (
+	"server/model"
+
+	"gorm.io/gorm"
+)
+
+type Telemetry struct {
+	model.BaseModel
+	MetricName  DeploymentMetric `gorm:"type:string"`
+	MetricValue string
+	Step        string
+}
+
+// Getter Setters for each DeploymentMetric
```

Comment:
Is it a Getter or Setter?

Reply:
SetMetric is the Setter and Metric is the Getter function